docs/reference-guides/interactivity-api/core-concepts/using-typescript.md

```diff
@@ -525,6 +525,8 @@ However, there are a couple of things to consider when using asynchronous action
 
     To fix this, you can combine the use of `yield*` with the `typed()` helper, which converts the promise type into an iterator that can be correctly inferred by TypeScript when used in conjunction with `yield*`.
 
+    _**Note**: The `typed` function will be included in WordPress 6.7._
```

Comment:
When WordPress 6.7 is released, we should update this note to something like this: `The typed function was included in WordPress 6.7.`